src/Bundle/Resources/config/services/field_types.xml

```diff
@@ -22,6 +22,13 @@
         </service>
         <service id="sylius.grid_field.datetime" alias="Sylius\Component\Grid\FieldTypes\DatetimeFieldType" />
```

Comment:
Normally it has been flipped back. I have to check.

Reply:
I confirmed, I think your branch is not up to date.

Reply:
I updated the config files